src/test/java/lithium/university/TwitterResourceTest.java

```diff
@@ -0,0 +1,69 @@
+package lithium.university;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TwitterResourceTest {
+    private TwitterRetrieve twitterRetrieveTest;
+    private TwitterPublish twitterPublishTest;
+
+
+    @Before
+    public void init() {
+        twitterRetrieveTest = Mockito.mock(TwitterRetrieve.class);
+        twitterPublishTest = Mockito.mock(TwitterPublish.class);
+    }
+
+    private Status mockStatus() {
+        return Mockito.mock(Status.class);
+    }
+
+    @Test
+    public void testResourceGetTimeline() throws TwitterException {
+        TwitterResource twitterResourceTest = new TwitterResource(twitterRetrieveTest, twitterPublishTest);
+        List<Status> fakeList = new ArrayList<>();
+        fakeList.add(mockStatus());
+
+        Mockito.when(twitterRetrieveTest.retrieveFromTwitter(Mockito.any(Twitter.class), Mockito.anyInt())).thenReturn(fakeList);
+
+        Object o = twitterResourceTest.getHomeTimeline();
+        if (!(o instanceof Tweet)) {
```

Comment:
better check would be Assert.true(o instanceof Tweet).  You should also check some data to verify the tweet returned is the expected tweet as opposed to just outputting it to command line.